system/Commands/Database/CreateDatabase.php

```diff
@@ -95,6 +95,14 @@ public function run(array $params)
 
             // Special SQLite3 handling
             if ($db instanceof Connection) {
+
+                if (!extension_loaded('sqlite3')) {
+                    CLI::error("extension [sqlite3] not loaded. Uncomment it in the php.ini file", 'light_gray', 'red');
+                    CLI::newLine();
+
+                    return;
+                }
```

Comment:
I don't accept this in here. It is only for SQLite3, so it should not in this class.